src/utils/fetchData.js

```diff
@@ -1,19 +1,21 @@
 import queryPerfData from '@mozilla-frontend-infra/perf-goggles';
-import { BENCHMARKS, CONFIG } from '../config';
+import linux64 from '../configuration/js-team/linux64';
 import prepareData from './prepareData';
 import { convertToSeconds } from './timeRangeUtils';
+import DASHBOARD_CONFIG from '../configuration/appDefaults';
+import platforms from '../configuration/js-team/index';
 
 const fetchData = async (platform, benchmark, range) => {
   const ALL_DATA = {};
 
-  const fetchIt = async (configUID, overview = false, timeRange = CONFIG.default.timeRange) => {
+  const fetchIt = async (configUID, overview = false, timeRange = DASHBOARD_CONFIG.timeRange) => {
     const includeSubtests = !overview;
-    const comparingBenchmarks = Object.keys(BENCHMARKS[configUID].compare);
+    const comparingBenchmarks = Object.keys(linux64.benchmarks[configUID].compare);
```

Comment:
```suggestion
    const comparingBenchmarks = Object.keys(config.views[view].benchmarks[configUID].compare);
```
I'm mentioning a variable `view` as used in this branch:
https://github.com/mozilla-frontend-infra/firefox-performance-dashboard/compare/newFetch

See if that branch is something you would like me to land. Even this new branch is not something I'm completely satisfied but it could help you.

Reply:
Hi, @armenzg Thanks for the review. I will get back to you in case of any issue when I get back to it.
Thanks for the help,
I am on other issues right now.